api/src/modules/data/services/data.service.ts

```diff
@@ -129,7 +129,7 @@ export class DataService {
 				throw err;
 			}
 			this.logger.error('Failed to get data from database', err.stack);
-			throw new InternalServerErrorException();
+			throw err;
```

Comment:
This exposes potentially sensitive information to the client side.
We want to log errors and just provide an indication of what is wrong.
For all we know this error could contain something like:
```
Failed to authenticate to graphql server, credentials used: username, password
```